recipe/cross_compile_support.sh

```diff
@@ -100,6 +100,12 @@ if [[ "${HOST_PLATFORM}" != "${BUILD_PLATFORM}" ]]; then
                 # filter packages from manifest down to what we need for cross-compilation
                 grep -E "cuda-(compat|cudart|cupti|driver|nvcc|nvml|nvprof|nvrtc|nvtx).*|lib(cu|npp|nvjpeg).*" rpms.txt > rpms_cc.txt
 
+                # the cuda-profiler-api package is not listed in the manifest, so we
+                # cannot generate it automatically
+                if [[ "${CUDA_COMPILER_VERSION}" == "11.8" ]]; then
+                    echo "cuda-profiler-api:11.8.86" >> rpms_cc.txt
```

Comment:
Wonder if there is a way to avoid hard-coding the version (as there could be patch bumps we want in the future)

Reply:
I'm sorry, but I don't know enough about how RPM channels work or the bandwidth to figure it out. If we were using the RPM package manager directly, then we could grep the results of a package search for "cuda-profiler-api-11-8".

Reply:
Yeah no worries. We can save for follow up

Reply:
This is exactly why the DEVELS mapping is there. You can have `cuda-profiler-api:cuda_nsight` to say that `cuda-profiler-api` version can be used from `cuda_nsight`

Reply:
I did consider stealing the version number from one of the packages that's actually listed in the manifest, but I wasn't sure that the version of `cuda-profiler-api` will always correspond to any of them. As opposed to "libcublas_devel" always corresponding to "libcublas".

I can make a corespondence between `cuda-profiler-api` and `cuda-nsight` if we all exepct that those two package versions will be linked.

Reply:
Sure let's give that a try and see how it goes. We can always follow up later as needed

Reply:
cuda-nsight doesn't exist in the aarch64 manifest, so I had to pick a different package to steal the version from